src/components/table.jsx

```diff
@@ -41,102 +34,141 @@ function Table({ Category, Title, search, grid }) {
+                             data-bs-content={data.notes}>emergency_home</span>}
       </div>
 
       {color === 'green' ?
         <>
           <div className="docs">
-            {data.documentation && <a aria-label="documentation" className="website-doc" href={data.documentation} />}
-            {data.recovery && <a aria-label="recovery documentation" className="recovery-doc" href={data.recovery} />}
+            {data.documentation &&
+              <a aria-label="documentation" className="website-doc"
+                 href={data.documentation}/>}
+            {data.recovery &&
+              <a aria-label="recovery documentation" className="recovery-doc"
+                 href={data.recovery}/>}
           </div>
 
-          <Methods methods={data?.methods} customSoftware={data["custom-software"]} customHardware={data["custom-hardware"]} />
-        </> :
-        <Contact contact={data.contact} />
+          <Methods methods={data?.methods}
+                   customSoftware={data['custom-software']}
+                   customHardware={data['custom-hardware']}/>
+        </>:
+        <Contact contact={data.contact}/>
       }
 
     </div>
   );
 }
 
+let staticTranslations = null; // Translations for table head
+
+function Head({category}) {
+  const t = useTranslation();
+
+  // Populate static translations once
+  if (!staticTranslations) {
+    staticTranslations = {
+      docs: t('docs'),
+      sms: t('sms'),
+      email: t('email'),
+      hardware: t('hardware'),
+      software: t('software'),
+    };
+  }
+
+  return html`
+    <div>${t(category)}</div>
+    <div>${staticTranslations.docs}</div>
+    <div>${staticTranslations.sms}</div>
+    <div>${staticTranslations.email}</div>
+    <div>${staticTranslations.hardware}</div>
+    <div>${staticTranslations.software}</div>
+  `;
+}
+
+function Methods({methods, customSoftware, customHardware}) {
 const mfaPopoverConfig = {
   html: true,
   sanitize: false,
   trigger: "hover focus"
 };
 
-function Methods({ methods, customSoftware, customHardware }) {
   useEffect(() => {
     [...document.querySelectorAll('.note')].map((el) => new Popover(el, {
       trigger: 'hover focus',
-      title: 'Exceptions & Restrictions'
-    }));
-
-
-    [...document.querySelectorAll('.custom-hardware-popover')].map((el) => new Popover(el, {
-      ...mfaPopoverConfig,
-      title: 'Custom Hardware 2FA'
+      title: 'Exceptions & Restrictions',
     }));
 
-    [...document.querySelectorAll('.custom-software-popover')].map((el) => new Popover(el, {
-      ...mfaPopoverConfig,
-      title: 'Custom Software 2FA'
-    }));
+    [...document.querySelectorAll('.custom-hardware-popover')].map(
+      (el) => new Popover(el, {
+        ...mfaPopoverConfig,
+        title: 'Custom Hardware 2FA',
+      }));
+
+    [...document.querySelectorAll('.custom-software-popover')].map(
+      (el) => new Popover(el, {
+        ...mfaPopoverConfig,
+        title: 'Custom Software 2FA',
```

Comment:
Should these titles be translated as well? We could use the `custom-hardware` and `custom-software` translations here.

Reply:
Yes, that would be preferable but I was unable to reliably update the title for the Popovers.
It might make more sense to replace them completely with standard HTML5 popovers.

Reply:
Fixed the translation by replacing the popovers with a custom popover HTML (and floating-ui as the popover JS)